docs/content/how-tos/rules/FL0084.md

```diff
@@ -0,0 +1,32 @@
+---
+title: FL0084
+category: how-to
+hide_menu: true
+---
+
+# EnsureTailCallDiagnosticsInRecursiveFunctions (FL0084)
+
+*Introduced in `0.23.8`*
```

Comment:
@webwarrior-ws let's change this to 0.24.2

Reply:
done